dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -156,72 +160,38 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
-
-                            pushChannelReceived = co_await channelManager.CreatePushNotificationChannelForApplicationAsync();
-
-                            PushNotificationTelemetry::ChannelRequestedByApi(S_OK, remoteId, usingLegacyImplementation);
-
-                            co_return winrt::make<PushNotificationCreateChannelResult>(
-                                winrt::make<PushNotificationChannel>(pushChannelReceived),
-                                S_OK,
-                                PushNotificationChannelStatus::CompletedSuccess);
-                        }
-                        else
-                        {
-                            winrt::check_hresult(hr);
-                        }
+                        auto appUserModelId{ PushNotificationHelpers::GetAppUserModelId() };
+                        THROW_IF_FAILED(CreateChannelWithRemoteIdHelper(appUserModelId, remoteId, channelInfo));
                     }
                     else
                     {
+                        // AppId is generated by PushNotificationLongRunningTask singleton
                         wil::unique_cotaskmem_string unpackagedAppUserModelId;
                         RegisterUnpackagedApplicationHelper(remoteId, unpackagedAppUserModelId);
-                        PushNotificationChannelManager channelManager{};
-                        winrt::PushNotificationChannel pushChannelReceived{ co_await channelManager.CreatePushNotificationChannelForApplicationAsync(unpackagedAppUserModelId.get()) };
+                        THROW_IF_FAILED(CreateChannelWithRemoteIdHelper(unpackagedAppUserModelId, remoteId, channelInfo));
 
                         auto notificationPlatform{ PushNotificationHelpers::GetNotificationPlatform() };
 
                         wil::unique_cotaskmem_string processName;
                         THROW_IF_FAILED(GetCurrentProcessPath(processName));
-                        THROW_IF_FAILED(notificationPlatform->RegisterLongRunningActivator(processName.get()));
+                        THROW_IF_FAILED(notificationPlatform->RegisterLongRunningActivatorWithClsid(processName.get(), GetComServerClsid()));
 
                         std::wstring toastAppId{ RetrieveNotificationAppId() };
                         THROW_IF_FAILED(notificationPlatform->AddToastRegistrationMapping(processName.get(), toastAppId.c_str()));
-
-                        PushNotificationTelemetry::ChannelRequestedByApi(S_OK, remoteId, usingLegacyImplementation);
-
-                        co_return winrt::make<PushNotificationCreateChannelResult>(
-                            winrt::make<PushNotificationChannel>(pushChannelReceived),
-                            S_OK,
-                            PushNotificationChannelStatus::CompletedSuccess);
                     }
+
+                    PushNotificationTelemetry::ChannelRequestedByApi(S_OK, remoteId, usingLegacyImplementation);
```

Comment:
I'm not sure how this compiles. This variable is no longer defined and is not apllicable to the flow either.